instructions-for-devs.md

```diff
@@ -0,0 +1,174 @@
+# Instructions for Developers
+
+This document contains `xud` setup instructions for developers and everyone with a need for more fine-grained control when running `xud`.
+
+## 1. Install
+
+First, clone the repository from GitHub and install dependencies. 
+
+Developers:
+```bash
+git clone https://github.com/ExchangeUnion/xud
+cd xud
+npm install
+```
+For everyone else, we recommend cloning the [latest release](https://github.com/ExchangeUnion/xud/releases):
+```bash
+git clone https://github.com/ExchangeUnion/xud -b 1.0.0-prealpha
```

Comment:
Here too I don't think we should specify a particular release or tag, although linking to our release page is good. I think the best approach is probably to publish our releases to npm, and then people can just do `npm install xud` to get the latest release. Until we have a proper release published to npm we can just leave this extra set of commands out I'd say.